elixir/apps/domain/lib/domain/application.ex

```diff
@@ -109,12 +77,40 @@ defmodule Domain.Application do
     end
   end
 
-  defp background_jobs_enabled? do
-    # The web and api applications also start Domain's supervision tree so that they may
-    # use the same database connection pool and other shared resources. We want to restrict
-    # some children to only run in the domain node.
-    # For now, we use BACKGROUND_JOBS_ENABLED to determine if we are running in the domain node.
-    # We could also use the node's name, but this doesn't work as well in dev/test environments.
-    Application.get_env(:domain, :background_jobs_enabled, false)
+  # TODO: Configure Oban workers to only run on domain nodes
+  defp oban do
+    [{Oban, Application.fetch_env!(:domain, Oban)}]
+  end
```

Comment:
Is the reason we're ok with this running on all nodes right now because there are no oban jobs at the moment?

Reply:
Yes and also I don't know if we need Oban to be running to _enqueue_ jobs.